torch/csrc/cuda/Stream.cpp

```diff
@@ -71,6 +71,7 @@ static PyObject* THCPStream_pynew(
 
   THCPStream* self = (THCPStream*)ptr.get();
   self->stream_id = static_cast<int64_t>(stream.id());
+  // NOLINTNEXTLINE(bugprone-signed-char-misuse)
```

Comment:
Ugh, this one is concerning...

Reply:
For uint8 is an alias of unsigned char...